### PR TITLE
Created the Contact Me Section (Forms - Validations - Footer)

### PR DESCRIPTION
**Pull request by fpsapc**
I have added get in touch section in the html file. Please you can do the CSS part for that section.

### CODE REVIEW

**Comment by hernandanielzamora** [comment]
Changes made to the CSS Files: 
 - Added style to the contact me form, including all the inputs, the textarea, and the button
 - Added style to the footer
 - Added "fontawesome" Icons to the footer

File: styles/style.css [hunk 1]
```diff
@@ -384,3 +384,77 @@ li {
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  height: 1.1em;
+}
+
+.footer-media li {
+  margin-right: 2%;
+}
```
Comment (Abenezer-Tilahun) [inline styles/style.css:460]: - [ ] As stated in the project requirement Kindly make sure that you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly adjust the font color, the background color , the text and the form input alignment and social icons just like the figma design below. 

- The Figma Design Look likes this : 
Capture

- Your Project Design Look likes this : 
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @fpsapc  , 

Great job for implemented  the suggested changes , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct Git Hub-Flow.
✔️ Good commit messages
✔️ Professional README file.

almost

## Required Changes ♻️
_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/style.css (hunk 1 above)
Comment (hernandanielzamora) [reply styles/style.css]: @Abenezer-Tilahun Hey! I´ve implemented the changes and now I´m waiting for a new review, I wanted to know if there's something you can help me with, I´ve been waiting for 3 hours now

**Review by Shisui6 — APPROVED** [review]
Hi @hernandanielzamora and @fpsapc, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Good design ✅
- HTML validation done correctly ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**